inheritance/src/main/java/inheritance/Library.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package inheritance;
+
+public class Library {
+    public boolean someLibraryMethod() {
+
+        return true;
+    }
+
+    // Restaurant class
+    public static class Restaurant {
+       static String name;
+       int numberStars;
+       int priceCategory;
+
+       // Restaurant constructor
+        public Restaurant(String name, int numberStars, int priceCategory) {
+            this.name = name;
+            this.numberStars = numberStars;
+            this.priceCategory = priceCategory;
+        }
+
+        // add reviews
+        public void addReview(Restaurant name, String body, String author, int numberStars) {
+            Review newReview = new Review(name, body, author, numberStars);
+
+            newReview.name.equals(Restaurant.name);
+
+            }
```

Comment:
These two `}` are indented too far, it makes it look like `Review` is nested inside of `Restaurant`